src/aotcompile.cpp

```diff
@@ -916,8 +915,7 @@ void *jl_get_llvmf_defn(jl_method_instance_t *mi, size_t world, char getwrapper,
         jl_llvm_functions_t decls;
         JL_LOCK(&codegen_lock);
         uint64_t compiler_start_time = 0;
-        int tid = jl_threadid();
-        if (jl_measure_compile_time[tid])
+        if (jl_atomic_load(&jl_measure_compile_time))
```

Comment:
Same here: this is the implementation of `code_llvm`, which is not part of compile time, as that is run at runtime.

Reply:
Same as above - maybe we can move this into a separate thread & separate PR?

This makes sense to me, but i'd like to do the change elsewhere.

Reply:
I've opened https://github.com/JuliaLang/julia/issues/41921 to discuss the remaining question here. I'll now resolve this thread.